src/table/__tests__/use-focus-navigation.test.tsx

```diff
@@ -143,7 +151,7 @@ describe('useTableFocusNavigation', () => {
 
     fireEvent.keyDown(getByTestId('0,2'), { key: 'ArrowLeft' });
 
-    waitFor(() => expect(focusFn.mock.calls).toEqual([['0,0'], ['0,2'], ['0,0']]));
+    expect(focusFn.mock.calls).toEqual([['0,0'], ['0,2'], ['0,0']]);
```

Comment:
Well, this is awkward. All these `waitFor` calls actually didn't work and the tests never waited for these assertions to be true.